bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -6,9 +6,9 @@ num_users_updated = 0
 batch_size = 10000
 
 puts "Processing..."
-
 User.where.not(school_info: nil).where(user_type: "teacher").find_in_batches(batch_size: batch_size) do |batch|
   ActiveRecord::Base.transaction do
+    exceptions = []
     batch.each do |user|
       UserSchoolInfo.create!(
```

Comment:
Because I suggested not rolling back the entire batch and because starting 2/28 new users will have a UserSchoolInfo, I think this needs to change to an ActiveRecord method that will create a UserSchoolInfo if none exist for the given User and SchoolInfo and otherwise do nothing.